test/factories/host_related.rb

```diff
@@ -299,6 +299,17 @@ def set_nic_attributes(host, attributes, evaluator)
       subnet
     end
 
+    trait :with_rootpass do
+      root_pass 'xybxa6JUkz63w'
```

Comment:
https://github.com/thoughtbot/factory_girl/blob/v4.5.0/GETTING_STARTED.md#sequences

i.e. not a static value, as changing values help keep people writing tests on their toes.